p5/io/local_data.py

```diff
@@ -52,27 +55,38 @@ def get_column(self,name):
 
 	def get_row(self,index):
 		"""
-		Returns an entire row when given the row index.
+		:param index: Index of the required row.
+		:type index: int
 
-		:param index: Name of the row
-		:type index: string
+		:returns: An entire row when given the row index.
+		:rtype: list
 		"""
-		for fragment in self.data:
-			if fragment[0] == index:
-				return fragment
+		return self.data[index]
 	def get_array(self):
 		"""
-		Returns the entire csv data as an multidimensional array.
+		:returns: the entire csv data as an multidimensional array.
+		:rtype: list
 		"""
 		return self.data
 
 
-def load_table(path):
+def load_table(path,mode):
 	"""
-	Calls Table class and returns a Table class object
+	Calls the Table class.
 	
 	:param path: Path to file
 	:type path: string
+
+	:param mode: Type of File csv/ssv.
+	:type mode: string
+
+	:returns: A Table object
+	:rtype: object
 	"""
-	table = Table(path)
+	if mode == "csv":
+		seperator = ','
+	if mode == "ssv":
+		seperator = ';'
+
```

Comment:
There is one more mode "tsv" "\t", add that too. :)

Reply:
I ll add that too 😀.